wicket-core/src/main/java/org/apache/wicket/markup/html/form/Form.java

```diff
@@ -757,7 +741,7 @@ public final boolean isSubmitted()
 	@Override
 	public final void onRequest()
 	{
-		onFormSubmitted(null);
+		onFormSubmitted(findSubmitter());
```

Comment:
This seems a bit problematic. We have failing Tests after upgrading from 10.2. to 10.4 and the reason seems to be that while the submitter is calculated the same as before, there is no check for visibility anymore (like in Line 810ff) and the code does run straight to Line 839 instead of Line 810 because the submitter is not null anymore when checking in line 801

Reply:
This might only influence Tests because in reality the user could not click a button that is not visible

Reply:
The best would be to send a PR with a fix and a test!

Reply:
https://github.com/apache/wicket/pull/1092